elfloader-tool/CMakeLists.txt

```diff
@@ -383,6 +403,17 @@ if(ElfloaderImageEFI)
             # EFI_SUBSYSTEM=0xa indicates that we're building an EFI application.
             " -Wl,-T ${linkerScript} -nostdlib -shared -Wl,-Bsymbolic,--defsym=EFI_SUBSYSTEM=0xa -Wl,--build-id=none"
     )
+elseif(ElfloaderPrecompile)
+    add_custom_command(
+      OUTPUT elfloader.o
```

Comment:
Could you put `elfloader.o` in a CMake variable, so we don't have the name hard-coded everywhere?

Reply:
What would the benefit be? We don't use a variable name for "kernel.elf", it's already an atom and the extra variable assignment is essentially redundant. The variable name would essentially just be `set(elfloader_o elfloader.o)` -> `${elfloader_o}` (`elfloader.o` is technically even a valid variable name).